scripts/update-translations.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Based on this confluence page:
+# https://confluence.dimagi.com/display/commcarehq/Internationalization+and+Localization+-+Transifex+Translations
+
+function abort () {
+    echo $1
+    echo "Aborting."
+    exit 1
+}
+
+has_local_changes=`git diff-index HEAD` && true
+if [[ $has_local_changes ]]
+then
+    abort "You have uncommitted changes in your working tree."
+fi
+
+branch=$(git rev-parse --abbrev-ref HEAD)
+if [[ $branch != 'master' ]]
+then
+    abort "You must be on master to run this command."
+fi
+
+if [[ ! `command -v tx` || ! -f ~/.transifexrc ]]
+then
+    echo "It looks like you haven't yet configured transifex."
+    echo "Please visit the wiki page for instructions on how to do so:"
+    echo "https://confluence.dimagi.com/display/commcarehq/Internationalization+and+Localization+-+Transifex+Translations"
+fi
+
+echo "Pulling translations from transifex"
+tx pull -f
+
+echo "Gathering all translation strings.  Note that this will probably take a while"
+./manage.py makemessages --all
+echo "Gathering javascript translation strings.  This will also probably take a while"
+./manage.py makemessages -d djangojs --all
+echo "Compiling translation files."
+./manage.py compilemessages
```

Comment:
Actually, I think I'm going to leave this in - it's much faster than the other two (~4s on my machine vs >1m for `makemessages`), and it could be confusing if someone's debugging translations locally using this script and doesn't actually see the translations.  Lemme know if you object.